core/pom.xml

```diff
@@ -183,13 +183,20 @@
             <id>default-test</id>
             <phase>none</phase>
           </execution>
+          <!-- The rdf-legacy module is an older version of RDF4J specifically targeting Android development. It isn't
```

Comment:
This explanation is fine, but why are we [including](https://github.com/inrupt/solid-client-java/blob/604de7e4f4931e911e71dc2ca6bfeef8f1905386/core/pom.xml#L150-L155) rdf-legacy in the dependencies in the first place?